packages/convenience/contracts/Convenience.sol

```diff
@@ -259,7 +260,7 @@ contract Convenience is Ownable  {
             return new uint256[](0);
         }
         uint256 countOpenVote = 0;
-        for (uint256 i = votesLength - 1; i >= 0; i--)
+        for (uint256 i = votesLength ; i > 0; i--)
```

Comment:
> for (uint256 i = votesLength - 1; i >= 0; i--)
This was leading underflow

Reply:
It did with the previous version but not this one because if `votesLength` is 0 it returns early
https://github.com/api3dao/api3-dao/blob/eb0e2cac9b58b17974bfce87733869f452a77bfc/packages/convenience/contracts/Convenience.sol#L258

Reply:
Even in the previous version, at the last iteration of the loop i.e i=0 it will call i-- and cause the i value to underflow for the next comparison of the loop.

Reply:
Ah right :+1: